optimum/neuron/utils/hub_neuronx_cache.py

```diff
@@ -238,23 +238,14 @@ def download_file_to_string(self, filename: str, limit: int = None):
         return s
 
 
-def get_hub_cache():
-    HUB_CACHE = "aws-neuron/optimum-neuron-cache"
-    custom_hub_cache = load_custom_cache_repo_name_from_hf_home()
-    if custom_hub_cache is not None and len(custom_hub_cache) > 0:
-        return custom_hub_cache
-    else:
-        return os.getenv("CUSTOM_CACHE_REPO", HUB_CACHE)
-
-
 def create_hub_compile_cache_proxy(
     cache_url: Optional[CacheUrl] = None,
     cache_repo_id: Optional[str] = None,
 ):
     if cache_url is None:
         cache_url = CacheUrl.get_cache_url()
     if cache_repo_id is None:
-        cache_repo_id = get_hub_cache()
+        cache_repo_id = get_hf_hub_cache_repo()
```

Comment:
I think there is a misunderstanding here: what is expected here is really a hub repo, not a local directory.
The local directory is the one provided by cacheurl.

Reply:
The misunderstanding is on my side: could you add a comment explaining that there is a priority between the several hub cache repos ?

Reply:
Solved offline